__e2e__/init.test.ts

```diff
@@ -198,7 +198,9 @@ test('init should contain Cocoapods instructions when pods are not installed aut
   createCustomTemplateFiles();
 
   const {stdout} = runCLI(DIR, ['init', PROJECT_NAME, '--skip-install']);
-  expect(stdout).toContain('Install Cocoapods');
+  if (process.platform === 'darwin') {
```

Comment:
I would make this whole test under a condition, because if we don't match condition we're just checking the same thing as in this test:
https://github.com/react-native-community/cli/blob/e1438f714407a9d590d5975ecd3c846b8d91ab21/__e2e__/init.test.ts#L109-L132

Reply:
Maybe let's add this check to the test you referenced?